src/main/resources/config/application-dev.yml

```diff
@@ -21,7 +21,7 @@ logging:
 
 spring:
     kafka:
-        bootstrap-servers: localhost:9093
```

Comment:
do you really need to change a port?

Reply:
yes, i spend 1 hour to understand why my code don't connect to Kafka
9093 it's incorrect port!
default port 9092